modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -1035,6 +1032,7 @@ public long leaseStartTime() {
     void destroyData(WriteBatch writeBatch) throws RocksDBException {
         writeBatch.delete(meta, lastAppliedIndexAndTermKey);
         writeBatch.delete(meta, lastGroupConfigKey);
+        writeBatch.delete(meta, leaseKey);
```

Comment:
Please fix `org.apache.ignite.internal.storage.rocksdb.instance.SharedRocksDbInstance#destroyTable` as well. Maybe there are other places, please be careful and ensure that you covered everything

Reply:
Thanks, fixed. I see no other usages of prefixes